### PR TITLE
[Search] Update changelog for "11.5.0-beta.5" release

### PR DESCRIPTION
**Pull request by ShivangiReja**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/search/tests.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ extends:
     CloudConfig:
       Preview:
         SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
-        Location: 'eastus2euap'
+        Location: 'westcentralus'
```
Comment (pallavit) [inline sdk/search/tests.yml:17]: Have the changes reached westcentralus?

**Review by pallavit — APPROVED** [review]

File: sdk/search/tests.yml (hunk 1 above)
Comment (ShivangiReja) [reply sdk/search/tests.yml:17]: Yes, all the changes are available in these regions

image

Comment (pallavit) [reply sdk/search/tests.yml:17]: You may need to move this back to EUAP when you start with GA swagger.

Comment (ShivangiReja) [reply sdk/search/tests.yml:17]: Some of my existing test cases where we use [SEARCH_COGNITIVE_KEY](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/search/test-resources.json#L203C10-L203C30) test resource was failing with [this](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3136875&view=logs&j=70ccd553-3427-5ec1-2920-06cca04de773&t=ac2c2ae1-f45f-5f65-4299-7a0ae8f99294&l=544) error:

> Error Message:Azure.RequestFailedException : Provided key is not a valid CognitiveServices type key for the region of your search service.
Status: 400 (Bad Request

Updating the `westcentralus` region works well for all the resources.

**Pull request merged**